include/swift/Runtime/InstrumentsSupport.h

```diff
@@ -43,6 +43,8 @@ SWIFT_RUNTIME_EXPORT
 void (*SWIFT_RT_DECLARE_ENTRY _swift_release_n)(HeapObject *object, uint32_t n);
 SWIFT_RUNTIME_EXPORT
 size_t swift_retainCount(HeapObject *object);
+SWIFT_RUNTIME_EXPORT
+std::atomic<bool> _swift_enableSwizzlingOfAllocationAndRefCountingFunctions_forInstrumentsOnly;
```

Comment:
It would fit the organization of the file slightly better if this were just above `swift_retainCount` instead of below.